polus-feature-extraction-plugin/src/main.py

```diff
@@ -0,0 +1,67 @@
+from utility import ConvertImage, Df_Csv_single
+import argparse, logging
+
+
+ # Initialize the logger
+logging.basicConfig(format='%(asctime)s - %(name)-8s - %(levelname)-8s - %(message)s',
+                    datefmt='%d-%b-%y %H:%M:%S')
+logger = logging.getLogger("main")
+logger.setLevel(logging.INFO)
+
+    # Setup the argument parsing
+def main():
+    logger.info("Parsing arguments...")
+    parser = argparse.ArgumentParser(prog='main', description='Everything you need to start a Feature Extraction plugin.')
+    parser.add_argument('--features', dest='features', type=str,
+                        help='Features to calculate', required=True)
+    parser.add_argument('--csvfile', dest='csvfile', type=str,
+                        help='Save csv as separate or single file', required=True)
+    parser.add_argument('--angleDegree', dest='angleDegree', type=int,
+                        help='Angle to calculate feret diameter', required=False)
+    parser.add_argument('--boxSize', dest='boxSize', type=int,
+                        help='Boxsize to calculate neighbors and feret diameter', required=False)
+    parser.add_argument('--intDir', dest='intDir', type=str,
+                        help='Intensity image collection', required=False)
+    parser.add_argument('--pixelDistance', dest='pixelDistance', type=int,
+                        help='Pixel distance to calculate the neighbors touching cells', required=False)
+    parser.add_argument('--segDir', dest='segDir', type=str,
+                        help='Segmented image collection', required=True)
+    parser.add_argument('--outDir', dest='outDir', type=str,
+                        help='Output collection', required=True)
+
+    # Parse the arguments
+    args = parser.parse_args()
+    features = args.features.split(',')#features
+    logger.info('features = {}'.format(features))
+    csvfile = args.csvfile#csvfile
+    logger.info('csvfile = {}'.format(csvfile))
+    angleDegree = args.angleDegree#angle to calculate feret diameter
+    logger.info('angleDegree = {}'.format(angleDegree))
+    boxSize = args.boxSize#box size to calculate neighbors and feret diameter
+    logger.info('boxSize = {}'.format(boxSize))
+    intDir = args.intDir#intensity image
+    logger.info('intDir = {}'.format(intDir))
+    pixelDistance = args.pixelDistance#pixel distance to calculate neighbors
+    logger.info('pixelDistance = {}'.format(pixelDistance))
+    segDir = args.segDir#segmented image
```

Comment:
Make sure to check if there is an `images` subfolder, and change the path to the subfolder.